forloop_modules/function_handlers/api_handlers.py

```diff
@@ -221,7 +221,7 @@ def make_form_dict_list(self, *args, node_detail_form=None):
         fdl.entry(name="cookies", text="", input_types=["dict"], row=4)
         fdl.label("Data")
         fdl.entry(name="filename", text="", input_types=["str"], required=True, row=5)
-        fdl.button(function=self.open_data_file, function_args=node_detail_form, text="Look up file", enforce_required=False)
+        fdl.button(function=self.open_data_file, function_args=node_detail_form, text="Look up file", enforce_required=False, name="lookup_txt_csv_file")
```

Comment:
I suggest changing it to "lookup_file", the suffixes can differ + be added/removed as we go, so it doesnt make much sense to me unless there would be 2 buttons in one handler

Reply:
you are right, but this seems like a good way how to transfer which file type should be loaded on cloud